docs/core/install/linux-fedora.md

```diff
@@ -12,32 +12,20 @@ ms.date: 01/06/2021
 
 [!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+For more information on installing .NET without a package manager, see one of the following articles:
 
-## Install .NET 5.0
-
-[!INCLUDE [linux-prep-intro-generic](includes/linux-prep-intro-generic.md)]
```

Comment:
.NET 5.0 is part of the default repositories so the preamble and the microsoft-repo commands are not required.